packages/sdk-middleware-auth/test/build-requests.spec.js

```diff
@@ -31,6 +31,28 @@ describe('buildRequestForClientCredentialsFlow', () => {
     })
   })
 
+  it('parses a host that ends with \'/\'', () => {
```

Comment:
:man_facepalming: what's the point then of having the commit hooks? 😅

Try to avoid then, unless it's really necessary please.